app.js

```diff
@@ -62,35 +94,125 @@ function startGame() {
 }
 
 function Step() {
-  renderEnemy(enemy1);
-  renderEnemy(enemy2);
-  renderEnemy(enemy3);
+  renderEnemy(enemyHealth50text, healthBar50text, enemyHealth50, healthBar50, enemy1);
+  renderEnemy(enemyHealth100text, healthBar100text, enemyHealth100, healthBar100, enemy2);
+  renderEnemy(enemyHealth200text, healthBar200text, enemyHealth200, healthBar200, enemy3);
   renderBullet (bullet);
   enemy1.x = moveEnemy(enemy1.x, enemy1Velocity);
   enemy2.x = moveEnemy(enemy2.x, enemy2Velocity);
   enemy3.x = moveEnemy(enemy3.x, enemy3Velocity);
+  enemyHealth50.x = moveEnemy(enemyHealth50.x, enemyHealth50Velocity);
+  enemyHealth100.x = moveEnemy(enemyHealth100.x, enemyHealth100Velocity);
+  enemyHealth200.x = moveEnemy(enemyHealth200.x, enemyHealth200Velocity);
+  enemyHealth50text.x = moveEnemy(enemyHealth50text.x, enemyHealth50textVelocity);
+  enemyHealth100text.x = moveEnemy(enemyHealth100text.x, enemyHealth100textVelocity);
+  enemyHealth200text.x = moveEnemy(enemyHealth200text.x, enemyHealth200textVelocity);
+  gutling.x = moveEnemy(gutling.x, gutlingVelocity);
   bullet.x += bullet.velocityX;
   bullet.y += bullet.velocityY;
   moveRocket();
   checkEnemyShipCollision(enemy1);
   checkEnemyShipCollision(enemy2);
   checkEnemyShipCollision(enemy3);
+  ammo.innerHTML = `Gutling Ammo: ${gutling.ammo} <br> Rocket Ammo: ${rocket.ammo}`;
```

Comment:
I suggest to rename `ammo` to `ammoElement`.
So we understand it's HTML Element, and not a Number.